wiki/Gameplay/Accuracy/de.md

```diff
@@ -0,0 +1,75 @@
+
+### ![](/wiki/shared/mode/mania.png) osu!mania
+
+![Genauigkeit = (50 \* Anzahl der 50er + 100 \* Anzahl der 100er + 200 \* Anzahl der 200er + 300 \* Anzahl der 300er + 300 \* Anzahl von MAX) / 300(Anzahl der 0er + Anzahl der 50er + Anzahl der 100er + Anzahl der 200er + Anzahl der 300er + Anzahl von MAX)](img/accuracy_mania.png "Genauigkeitsformel für osu!mania")
+
+In osu!mania wird die Genauigkeit ähnlich wie bei [osu!](#-osu!) berechnet.
+
+## Leistungsgrafik
+
+![Leistungsgrafik](img/performance_graph.png "Leistungsgrafik")
+
+Die Leistungsgrafik ist ein Diagramm, das die Leistung des Spielers (basierend auf seinem Lebensbalken) im Verlauf eines Spiels (Zeit) anzeigt. Zusätzliche Informationen können angezeigt werden, wenn man den Mauszeiger im Spiel darüber bewegt.
+
+*Hinweis: Die zusätzlichen Informationen können nur nach dem Abspielen einer Beatmap oder dem Ansehen eines Replays angezeigt werden. Nach dem Verlassen des [Ergebnisbildschirms](/wiki/Interface#ranking-screen) werden diese Informationen nicht gespeichert.*
+
+### Genauigkeit
+
+Wenn man den Mauszeiger über die Leistungsgrafik bewegt, wird ein Tooltip mit den Bewertungen " Error" und " Unstable Rate" angezeigt.
+
+Aufgrund der Art und Weise, wie die Mods [DT](/wiki/Game_modifier/Double_Time) (Double Time) und [HT](/wiki/Game_modifier/Half_Time) (Half Time) implementiert sind, werden die Werte für Error und Unstable Rate mit demselben Faktor wie der Song multipliziert. Um die wahren Werte zu erhalten, wenn man mit der DT-Mod spielt, teilt man die Ergebnisse durch 1,5. In ähnlicher Weise multipliziert man die Ergebnisse mit 1,33, wenn man mit der HT-Mod spielt.
+
+#### Error
+
+`Error` zeigt immer zwei Werte an, die angeben, wie weit die frühen Treffer im Durchschnitt entfernt waren und wie weit die späten Treffer im Durchschnitt entfernt waren. Je höher der [Overall Difficulty](/wiki/Beatmapping/Overall_difficulty)-Wert der Beatmap ist, desto niedriger müssen die Fehlerwerte sein, um beim Spielen der Beatmap gut abzuschneiden.
+
+#### Unstable rate
+
+`Unstable Rate` steht für die Beständigkeit des Timings der Treffer, wobei niedrigere Zahlen besser sind (Top-Spieler erreichen oft Werte unter 100). Zu beachten ist, dass der Wert die Beständigkeit und nicht die Genauigkeit misst, das heißt, 15ms zu früh zu treffen ist dasselbe wie rechtzeitig zu treffen. 
+Die Formel ist im Wesentlichen die Standardabweichung der Trefferfehler (in Millisekunden), multipliziert mit 10. [Beispielcode](https://gist.github.com/peppy/3a11cb58c856b6af7c1916422f668899) ist als Referenz verfügbar und zeigt, wie osu-stable die Unstable Rate-Werte berechnet.
+
+### Spin
+
+*Hinweis: Spin wird nur für den [osu! Spielmodus](/wiki/Spielmodus/osu!) verwendet.*
+
+Zusätzlich zur Genauigkeit werden in der gleichen Meldung auch einige Informationen zu Spinnern angezeigt. <!-- This line could use some more information on what that information is, how it's calculated, what it means, etc. etc. -->
+
+#### Speed
```

Comment:
`Geschwindigkeit` instead of speed?

Reply:
left it as `Speed` since its also not translated in game and might be confusing cause i also didnt translate `Error` or `Unstable Rate` for the same reason